apps/kyb-app/src/components/organisms/UIRenderer/elements/JSONForm/components/FieldTemplate/FieldTemplate.tsx

```
@@ -18,14 +20,21 @@ export const FieldTemplate = (props: FieldTemplateProps) => {
   const { state } = useDynamicUIContext();
   const { payload } = useStateManagerContext();
   const { definition } = useJSONFormDefinition();
+  const inputIndex = useMemo(() => {
+    const index = getInputIndex(props.id || '');
+
+    return isNaN(index as number) ? null : index;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)